compiler/hash-typecheck/src/new/diagnostics/error.rs

```diff
@@ -6,7 +6,7 @@ use hash_reporting::{
 use hash_source::location::SourceLocation;
 use hash_types::new::terms::TermId;
 
-use crate::new::{environment::tc_env::WithTcEnv, ops::common::CommonOps};
+use crate::new::{environment::tc_env::WithTcEnv, ops::common_ops::CommonOps};
```

Comment:
overly verbose naming in my opinion `ops::<...>_ops`

Reply:
Hmm, fine, i will change it. I initially did this cause it was annoying to have files called `tuple.rs`